\subsection{Astrobiology Methods}
\label{sec:Astrobiology Methods}

\subsubsection{Vacuum Chamber Testing}


The pumps, along with the temperature, humidity and pressure sensors and tubing will undergo extensive thermal vacuum testing in the range of \SIrange{-3}{25}{\celsius}, with a pressure range of \SIrange{0}{10}{\milli\bar}. The vacuum chamber tests will run for approximately \SIrange{8}{15}{\hour}, to replicate conditions from the previous flights. In the past~\cite{SORA}, the pumps were subjected to a temperature range of \SIrange{-30}{50}{\celsius} during integration and ran again for \SI{8}{\hour} afterwards and prior to flight. The pumps have proven to function in extremem environments without an issues. 


\subsubsection{Pre-Flight Preparation}


The clean box, collection containers and tubing will be autoclaved. All tools used in the assembly of the clean box will either be autoclaved or soaked in a \SI{70}{\percent} ethanol solution inside of a clean room. Each person who enters the clean room will be garbed in a lab coat, goggles, hair net and latex gloves after thoroughly washing their hands in a \SI{70}{\percent} ethanol solution. 

%We need to reword this to fit what Dr. Pattison wants us to do now that we have her onboard

%The \SI{15}{\percent} glycerol solution will poured into each container, the lid was sealed with silicone gasket maker and the tubing was inserted and gasket sealed into the container lids. Each lid had two holes, one that led to the inside of the clean box to allow for pressure to be released from inside the container and outgassed through the valve embedded in the box, while the other hole passed through the clean box lid to allow the tubing to connect to the pump or solenoid only in the case of the control tubing. The lid to the clean box was sealed with silicone gasket maker, the box was then mounted onto the payload and the tube from the control container was clamped to the dedicated control solenoid, while the sample collecting tube was passed through the other solenoid and connected to the pump. A final piece of tubing was connected from the intake valve on the pump to the outside of the payload, after a \SI{70}{\percent} ethanol solution was run through the pump several times. The end of the tube was bent, and zip tied. The payload was closed and remained in the clean room until it was ready for transport to Fort Sumner.  At the launch site, the zip tie was cut and the exposed inner and outer tubing was swabbed with an alcohol pad approximately \SI{4.5}{\hour} prior to launch. 



\subsubsection{Post Flight Procedures}

%Reword this section too to match what we plan to do after we recover it.  Include first that we will put the cleanbox in an iced cooler.

%The payload was successfully retrieved on September~6,~2017. The intact clean box was removed and placed inside of a cooler with ice, transported to The University of Houston and placed in cold storage at \SI{4}{\celsius}. All equipment used in the filtration process was either autoclaved or taken from previously unopened sanitized packaging. The autoclaved, pre-sanitized items and the clean box were washed in a \SI{70}{\percent} ethanol solution before they were placed inside a SterilGARD e3 Class II Biological Safety Cabinet. The Cabinet has a laminar flow air barrier and UV lights built into the ceiling for decontaminating the workspace prior to use. Both the control and sample collection solutions were vacuum filtered through a Fluropore membrane filter (\SI{13}{\milli\meter}; \num{0.22} micron) to collect specimens on the filter surface.  The filters were packaged for shipment to RTL Genomics~\cite{RTL} for 16S ribosomal RNA sequencing. All post flight sanitation and sample and control filtration procedures took place under the supervision of Professor Donna Pattison from the Department of Biology and Biochemistry at The University of Houston.


%\subsubsection{Ribosomal RNA Sequencing and Data Analysis}
%
%
%
%We sent our filtered control and experimental samples to RTL Genomics in Lubbock, Texas for ribosomal RNA sequencing and data analysis. We selected the 926wF  (``AAACTYAAAKGAATTGRCGG'') and 1392R (``ACGGGCGGTGTGTRC***'') primers for the sequencing procedure. These primers are designed to amplify 16S RNA from bacterial, archaeal and eukaryotic ``universal'' samples. The samples were amplified using a two step PCR procedure~\cite{lemon}. Samples were sequenced on an Illumina MiSeq~\cite{illumina} 2x300 flow cell at 10~pM and sequenced at RTL Genomics using standard sequencing procedures~\cite{Microbial rRNA sequencing}.

